gyp_xwalk

```diff
@@ -351,6 +351,9 @@ if __name__ == '__main__':
     args.append('-Dnotifications=1')
     args.append('-Drelease_unwind_tables=0')
 
+  if sys.platform == 'win32':
```

Comment:
Weird. It should work. I just tried it too. It works for me. The following is the snapshot:
![widevine_test](https://cloud.githubusercontent.com/assets/4221210/12570606/0472d5ee-c414-11e5-9608-c284d27f5187.PNG)
Here is my steps:
1. Sync the code to the newest.
2. <code>python gyp_xwalk.py -Dmediacodecs_EULA=1</code>
3. <code>ninja -C ../out/Debug xwalk</code>
4. <code>../out/Debug/xwalk.exe  --remote-debugging-port=9222 http://10.239.14.9/widevine/?dash\;asset=assets/car_segmenttemplate.mpd</code>

Note: The test video is set up on my local server since http://shaka-player-demo.appspot.com/?dash;asset=assets/car_segmenttemplate.mpd is very slow. I copied the content from that site.

Then, everything works well. You don't need to do any additional operation.

>I get a message saying the Widevine CDM is not available. Do the Widevine DLLs need to be in some special directory for things to work?

Do you notice any log info show that 'File path path/to/widevinecdmadapter.dll doesn't exist'? If yes, it means you didn't successfully generate 'widevinecdmadapter.dll' files.
In our code, in <code>xwalk.gyp</code>, <code>'../third_party/widevine/cdm/widevine_cdm.gyp:widevinecdmadapter'</code> will produce <code>widevinecdmadapter.dll</code> which located in the same directory with binary code. Such as, for debug build, it will locate in <code>src/out/Debug/widevinecdmadapter.dll</code>. And we don't need to move it to any special path (Our code have ensured that <code>xwalk::FILE_WIDEVINE_CDM_ADAPTER</code> path is the path where <code>widevinecdmadapter.dll</code> located).